doc/api/stream.md

```diff
@@ -2886,8 +2886,38 @@ readable.on('data', (chunk) => {
 
 #### Piping to writable streams from async iterators
 
-In the scenario of writing to a writable stream from an async iterator, ensure
-the correct handling of backpressure and errors.
+In the scenario of writing to a writable stream from an async iterator, the
+correct handling of backpressure and errors must be ensured.
+[`stream.pipeline()`][] conveniently abstracts away the handling of
```

Comment:
Another optional suggestion, but I try to avoid adverbs if they're not necessary.

```suggestion
[`stream.pipeline()`][] abstracts away the handling of
```

Reply:
Done. Thanks for the tip!